src/text.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     homePage: 'Home Page',
     english: 'English',
     arabic: 'Arabic',
+    error: 'Bad request !!',
```

Comment:
Might be good to have other errors too (e.g. `Not Found`)

Also maybe drop the exalamation marks?